composer.json

```diff
@@ -11,22 +11,22 @@
     ],
     "require": {
         "php": ">=8.0.5",
-        "bref/bref": "^1.3",
+        "bref/bref": "^1.3 || ^2.0",
```

Comment:
I'm not so sure we can just bump this and have it work.

Reply:
has dependencies to symfony 6.x components, so we cannot bump the symfony on our side